### PR TITLE
fixed creation of start and end date of event ( #1754, #1772)

### PR DESCRIPTION
**Pull request by dashapenkova**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/components/Notifications/Messages.tsx [hunk 1]
```diff
@@ -7,6 +7,10 @@ export const emptyInput = (name?:string)=>{
     return name ? `Поле ${name} є обов'язковим` : `Поле є обов'язковим`;
 };
 
+export const incorrectStartTime = "Час початку події не може бути раніше поточного часу!";
+
+export const incorrectEndTime = "Час завершення події не може бути часу початку!";
```
Comment (christykobyletska) [inline src/components/Notifications/Messages.tsx:12]: I think, here one word is missing, maybe something like this: "Час завершення події не може бути **раніше** часу початку"

File: src/pages/Actions/ActionEvent/EventCreate/EventCreate.tsx [hunk 2]
```diff
@@ -105,12 +108,21 @@ export default function ({ onCreate, setShowEventCreateDrawer }: Props) {
     setShowEventCreateDrawer(false);
   }
 
+
```
Comment (christykobyletska) [inline src/pages/Actions/ActionEvent/EventCreate/EventCreate.tsx:111]: why did you add this empty line?

File: src/pages/Actions/ActionEvent/EventCreate/EventCreate.tsx [hunk 3]
```diff
@@ -208,14 +220,23 @@ export default function ({ onCreate, setShowEventCreateDrawer }: Props) {
       </Row>
       <Row justify="start" gutter={[12, 0]}>
         <Col md={24} xs={24}>
-          <Form.Item label="Дата початку" name="EventDateStart" className={classes.formItem} rules={[{ required: true, message: emptyInput() }]}>
+          <Form.Item label="Дата початку" name="EventDateStart" className={classes.formItem}
+            rules={[
+              { required: true, message: emptyInput() },
```
Comment (christykobyletska) [inline src/pages/Actions/ActionEvent/EventCreate/EventCreate.tsx:225]: please format this piese of code like this: 
```suggestion
              { 
                required: true, 
                message: emptyInput() 
              },
```

File: src/pages/Actions/ActionEvent/EventCreate/EventCreate.tsx [hunk 4]
```diff
@@ -224,9 +245,22 @@ export default function ({ onCreate, setShowEventCreateDrawer }: Props) {
       </Row>
       <Row justify="start" gutter={[12, 0]}>
         <Col md={24} xs={24}>
-          <Form.Item label="Дата завершення" name="EventDateEnd" className={classes.formItem} rules={[{ required: true, message: emptyInput() }]}>
+          <Form.Item label="Дата завершення" name="EventDateEnd" className={classes.formItem} 
+          rules={[
+            { required: true, message: emptyInput() },
```
Comment (christykobyletska) [inline src/pages/Actions/ActionEvent/EventCreate/EventCreate.tsx:250]: as well as here:
Suggested change 
              { required: true, message: emptyInput() },
              { 
                required: true, 
                message: emptyInput() 
              },

File: src/components/Notifications/Messages.tsx (hunk 1 above)
Comment (dashapenkova) [reply src/components/Notifications/Messages.tsx]: oops, missed it. Thanks

**Review by VolodymyrKostiv — APPROVED** [review]
